test/common/storagelib.py

```diff
@@ -368,6 +368,39 @@ def doit():
                 raise
         self.browser.wait(doit)
 
+    # Creating and formatting a partition sometimes fails with "Device
+    # or resource busy" when the storage stack trips over its own feet
+    # during the process.  Let's do what the user would do and
```

Comment:
which is "swear and go back to MacOS"? :grin: :see_no_evil: 

More seriously, feels like an OK hack in our current position. Thanks!

Reply:
Yeah... oh, wipefs has --lock and even documents it to be used to coordinate with udevd... let me make this libblockdev PR...

Reply:
Hmm, flock will not protect against open returning EBUYS...